code/modules/organs/blood.dm

```diff
@@ -125,11 +125,23 @@ var/const/BLOOD_VOLUME_SURVIVE = 40
 
 		//Bleeding out
 		var/blood_max = 0
+		var/blood_loss_divisor = 30	//lower factor = more blood loss
+		//This 30 is the "baseline" of a cut in the "vital" regions (head and torso).
 		for(var/obj/item/organ/external/temp in organs)
```

Comment:
You could possibly cut down on this a lot by changing it to `for(var/obj/item/organ/external/temp in bad_external_organs)`